app/message-pin/client/pinMessage.js

```diff
@@ -1,42 +1,50 @@
 import { Meteor } from 'meteor/meteor';
 import { settings } from '../../settings';
 import { ChatMessage, Subscriptions } from '../../models';
+import toastr from 'toastr';
+import { TAPi18n } from 'meteor/tap:i18n';
 
 Meteor.methods({
 	pinMessage(message) {
 		if (!Meteor.userId()) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
 			return false;
-		}
-		if (!settings.get('Message_AllowPinning')) {
+		} else if (!settings.get('Message_AllowPinning')) {
```

Comment:
we dont need an `else` statement here, as we have a `return` inside each if

Reply:
@ggazzo I had done this PR in gsoc19 time and thus lost my old repository. I had made a new PR this year for the same issue. Sorry for the inconvenience.

#New PR #16056